xapian-core/include/xapian/queryparser.h

```diff
@@ -1066,6 +1066,20 @@ class XAPIAN_VISIBILITY_DEFAULT QueryParser {
 		      unsigned flags = FLAG_DEFAULT,
 		      const std::string &default_prefix = std::string());
 
+    /** Parse a math formula query.
+     *
+     * @param query_string	Math expression as std::string.
+     * @param flags		Zero or more QueryParser::feature_flag
+     * 		specifying what features the QueryParser should support.
+     * 		Combine multiple values with bitwise-or (|)
+     * 		(default FLAG_DEFAULT).
+     *	@param default_prefix	The default term prefix to use (default none.
+     *		For example, you can pass "A" when parsing an "Author" field.
+     */
+    Query parse_math_query(const std::string & query_string,
+			   unsigned flags = FLAG_DEFAULT,
+			   const std::string &default_prefix = std::string());
```

Comment:
Any thoughts on what and when flags and default_prefix would be useful in math query and indexing?

If we can't think any good usecase of flags and default_prefix, does it make sense to have them in function for sake of consistency with parse_query function?

Reply:
Right, `flags` parameter is not needed. I removed the usage. Please check another comment for a possible use case for flags.